### PR TITLE
Login signup

### PR DESCRIPTION
**Pull request by moath143**
Issue #12 
Login and Sign up pages are finished and @yasmeentayeh99  will complete the validation for the forms

### CODE REVIEW

File: src/components/input/Input.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+interface InputProps {
+  type: string;
+  placeholder: string;
+}
+
+
+const Input: React.FC<InputProps> = (props) => {
+  return (
+    <>
```
Comment (mossa-Sammer) [inline src/components/input/Input.tsx]: unneccssary fragment

File: src/components/input/Input.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import React from "react";
+
+
+interface InputProps {
+  type: string;
+  placeholder: string;
+}
+
+
+const Input: React.FC<InputProps> = (props) => {
```
Comment (mossa-Sammer) [inline src/components/input/Input.tsx]: Your are limiting the TextField API to only two props while you can use the interface provided by materail-ui for the text field

File: src/components/navbar/Navbar.tsx [hunk 3]
```diff
@@ -1,27 +1,46 @@
...
+import { NavbarList, Logo, SpanCounter, AnchorLink } from "./styleNavbar";
+import SearchIcon from "@mui/icons-material/Search";
+import FavoriteBorderOutlinedIcon from "@mui/icons-material/FavoriteBorderOutlined";
 
-const Navbar = () => {
+interface NavbarProps {
+  navColor: string;
+  counterColor: string;
+}
+//: React.FC<NavbarProps>
```
Comment (mossa-Sammer) [inline src/components/navbar/Navbar.tsx:10]: commented code

File: src/components/navbar/Navbar.tsx [hunk 4]
```diff
@@ -1,27 +1,46 @@
...
+const Navbar = (props: NavbarProps) => {
   return (
-    <Grid sx={{margin: 0, padding: '10px 0'}} container spacing={3}>
```
Comment (mossa-Sammer) [inline src/components/navbar/Navbar.tsx:6]: Please use styled components

**Review by mossa-Sammer — CHANGES_REQUESTED** [review]

**Review by yasmeentayeh99 — APPROVED** [review]

**Pull request merged**